pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -362,6 +410,22 @@ func (plugin *cniNetworkPlugin) deleteFromNetwork(network *cniNetwork, podName s
 	return nil
 }
 
+func (plugin *cniNetworkPlugin) SetPodCNIResult(podSandboxID kubecontainer.ContainerID, cniResult *cnicurrent.Result) {
```

Comment:
This doesn't need to be exported either.

Reply:
agree